src/test/java/io/xlate/edi/internal/stream/StaEDIStreamReaderTest.java

```diff
@@ -355,6 +355,58 @@ void testGetDelimitersX12_WithISX_00501() throws EDIStreamException {
         assertEquals(1, delimiterExceptions, "Unexpected exceptions");
     }
 
+    @Test
+    void testGetDelimitersX12_WithISX_00501_Invalid_Repitition() throws EDIStreamException {
+        EDIInputFactory factory = EDIInputFactory.newFactory();
+        ByteArrayInputStream stream = new ByteArrayInputStream((""
+                + "ISA*00*          *00*          *ZZ*ReceiverID     *ZZ*Sender         *200711*0100*V*00501*000000001*0*T*:~"
+                + "ISX*\\~"
+                + "GS*FA*ReceiverDept*SenderDept*20200711*010015*1*X*005010~"
+                + "ST*997*0001*005010X230~"
+                + "SE*2*0001~"
+                + "GE*1*1~"
+                + "IEA*1*000000001~").getBytes());
+
+        EDIStreamReader reader = factory.createEDIStreamReader(stream);
+        Map<String, Character> expected = new HashMap<>(5);
+        expected.put(Delimiters.SEGMENT, '~');
+        expected.put(Delimiters.DATA_ELEMENT, '*');
+        expected.put(Delimiters.COMPONENT_ELEMENT, ':');
+        expected.put(Delimiters.REPETITION, '^');
```

Comment:
In conjunction with the other suggestion:
```suggestion
        //expected.put(Delimiters.REPETITION, '^');
```